docs/admin/deployment.md

```diff
@@ -4,120 +4,123 @@
+  marketplace-restserver:
+    db_user: <username>   # default is 'user'
+    db_password: <password> # default is 'passwd'
+    azure_storage: # set this value if you want use Azure Blob as storage
+      storage_account: <storage_account>
+      connection_strings:
+      - '<connection_string>'
 
-3. Config **marketplace-restserver**.
+  marketplace-webportal:
+    marketplace_api_uri: https://<openpai cluster>/marketplace/api
 
-   Add following config in **service-configuration.yaml** to config `marketplace-restserver` service: 
+  webportal:
+    plugins:
+    - id: marketplace
+      title: Marketplace
+      uri: https://<openpai cluster>/marketplace/plugin.js
 
-   ```
-   marketplace-restserver:
-     db_user: <username>   # default is 'user'
-     db_password: <password> # default is 'passwd'
-     azure_storage:
-       storage_account: <storage_account>
-       connection_strings:
-       - '<connection_string>'
-   ```
+  ## other configs ...
+  ```
 
-   For more details about the config of **marketplace-restserver**, please refer to [marketplace-restserver/config](https://github.com/microsoft/pai/tree/master/src/marketplace-restserver/config), for the **azure_storage** config, please refer to [Data Management](../user/data_management.md).
+  TODO: does the file value correctly and indicate properties required or optional 
 
+  - `cluster.common.marketplace`: this config is the switch of marketplace, if this value is set true, marketplace will be enabled in OpenPAI, if this value is false or not set, marketplace will be disabled in OpenPAI.
 
-4. Config **marketplace-webportal**.
+  - `marketplace-db`: including `user, passwd and data-path` values. These values controls the basic info of postgresql database marketplace used.
 
-   Add following config in **service-configuration.yaml** to config `marketplace-webportal` service: 
+  - `marketplace-restserver`: including `db_user, db_password and azure_storage`. These values are used when starting [Marketplace rest server](https://github.com/microsoft/pai/tree/master/src/marketplace-restserver/config). The first two values should be aligned with values set in `marketplace-db`. The last value `azure_storage` is not required, if this value exists, rest server will initialize it into database as an available [azure blob storage](../user/data_management.md).
 
-   ```
-   marketplace-webportal:
-     marketplace_api_uri: https://<openpai cluster>/marketplace/api
-   ```
+  - `marketplace-webportal.marketplace_api_uri`: this value is set with Marketplace rest server api url. If pylon is deployed in OpenPAI, marketplace_api_uri could be set as `https://<openpai_cluster_ip>/marketplace/api`. If pylon is not deployed in OpenPAI, then marketplace_api_uri should be set as `https://<master_node_ip>:<9292>`. (TODO: if pylon is not set, the request will failed because of cors?)
```

Comment:
```suggestion
  - `marketplace-webportal.marketplace_api_uri`: this value is set with Marketplace rest server api url. If pylon is deployed in OpenPAI, marketplace_api_uri could be set as `https://<openpai_cluster_ip>/marketplace/api`. If pylon is not deployed in OpenPAI, then marketplace_api_uri should be set as `http://<master_node_ip>:<9292>`. (TODO: if pylon is not set, the request will failed because of cors?)
```